src/OpenTelemetry.Exporter.Prometheus.HttpListener/PrometheusExporterHttpListenerMeterProviderBuilderExtensions.cs

```diff
@@ -0,0 +1,85 @@
+// <copyright file="PrometheusExporterHttpListenerMeterProviderBuilderExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using OpenTelemetry.Exporter;
+using OpenTelemetry.Exporter.Prometheus.HttpListener;
+using OpenTelemetry.Internal;
+
+namespace OpenTelemetry.Metrics
+{
+    public static class PrometheusExporterHttpListenerMeterProviderBuilderExtensions
+    {
+        public static MeterProviderBuilder AddPrometheusHttpListener(
+            this MeterProviderBuilder builder,
+            Action<PrometheusExporterOptions> configureExporterOptions = null,
```

Comment:
Where is `PrometheusExporterOptions` class defined? Why we need two separate options class?